src/NuGet.Core/NuGet.Protocol.Core.Types/Model/PackageSearchMetadataBuilder.cs

```diff
@@ -123,5 +126,10 @@ public static IPackageSearchMetadata WithVersions(this IPackageSearchMetadata me
                 .WithVersions(AsyncLazy.New(valueFactory))
                 .Build();
         }
+
+        public static string GetFullName(this IPackageSearchMetadata iPackageSearchMetadata)
```

Comment:
Also, it looks like this is only used in one place so can we remove this method?

---
In reply to: [94629865](https://github.com/NuGet/NuGet.Client/pull/1088#discussion_r94629865) [](ancestors = 94629865)